mssqlcli/mssqlclioptionsparser.py

```diff
@@ -0,0 +1,159 @@
+import argparse
+import mssqlcli
+import os
+
+from .config import config_location
+
+MSSQL_CLI_USER = u'MSSQL_CLI_USER'
+MSSQL_CLI_PASSWORD = u'MSSQL_CLI_PASSWORD'
+MSSQL_CLI_DATABASE = u'MSSQL_CLI_DATABASE'
+MSSQL_CLI_SERVER = u'MSSQL_CLI_SERVER'
+MSSQL_CLI_ROW_LIMIT = u'MSSQL_CLI_ROW_LIMIT'
+MSSQL_CLI_RC = u'MSSQL_CLI_RC'
+
+
+def initialize():
```

Comment:
We have two public methods that do the same thing: initialize and get_parser. Can we have a single method: create_parser?